projects/aca-shared/src/lib/components/info-drawer/info-drawer.component.spec.ts

```diff
@@ -57,6 +57,38 @@ describe('InfoDrawerComponent', () => {
       ])
   };
 
+  const mockNode = {
```

Comment:
Same here for node ids and usage of defined properties

Reply:
Updated.